OrbitGl/App.cpp

```diff
@@ -270,8 +269,15 @@ void OrbitApp::PostInit() {
   }
 
   if (!options_.grpc_server_address.empty()) {
-    grpc_channel_ = grpc::CreateChannel(options_.grpc_server_address,
-                                        grpc::InsecureChannelCredentials());
+    grpc::ChannelArguments channel_arguments;
+    // The default receive message size is 4mb. Symbol data can easily be more
+    // than this. This is set to an arbitrary size of 400mb, which seems
+    // to be enough and leaves some headroom. As an example, a 1.1gb .debug
+    // symbols file results in a message size of 88mb.
+    channel_arguments.SetMaxReceiveMessageSize(400 * 1024 * 1024);
```

Comment:
What happens if we a message is bigger than the limit?  This limit should probably be exposed to the user  (in Params.h) to be safe so that if ever there is an exceptionally big debug symbols file, a user could still use Orbit.

Reply:
What happens is there will be an error displayed to the user that the grpc message is too large. 

I like the idea of a user definable parameter, but I don't know if we should continue to use `Params.h` because `GParams` is a global. 
I am going to take this to the chat room, maybe other people have opinions